.dockerignore

```diff
@@ -4,4 +4,4 @@
 out
 storybook-static
 .gitsecret
-.git
\ No newline at end of file
+.git
```

Comment:
I need to figure out how to configure my editor to not make changes like this automatically